docs/application/dotnet/getting-started/app-filtering.md

```diff
@@ -394,6 +395,7 @@ The following tables show the available requirements for an application package.
 | `http://tizen.org/feature/speech.control` | Specify this key, if the application requires the voice control feature. | 2.4   |
 | `http://tizen.org/feature/speech.recognition` | Specify this key, if the application requires the speech recognition (STT) feature. | 2.2.1 |
 | `http://tizen.org/feature/speech.synthesis` | Specify this key, if the application requires the speech synthesis (text to speech, TTS) feature. | 2.2.1 |
+| `http://tizen.org/feature/storage.external` | Specify this key, if the application requires the external storage feature. | 5.5   |
```

Comment:
Please make this small change:

Specify this key, if the application requires an external storage feature.

Reply:
@divyk How about getting the consistency with other sentences?

Reply:
Ok, I agree, we can retain the original sentence. Thanks.